core/src/test/python/compare_model_test.py

```diff
@@ -190,6 +190,60 @@ def testCompareModelInvalidFile(self):
 
         self.assertNotEqual(return_code, 0)
 
+    def testDeleteModelAppDeploymentsRemoved(self):
```

Comment:
Is this like a double negative?  Delete ... Removed.  Please add a comment, or change the name of the method to clarify the intent.